<?xml version="1.0" ?>
<launch>
	<node name="PointPlot" pkg="robot_task" type="point_plot" output="screen" >

      <param name="radius" value = "0.02" />
      <param name="heigh" value = "0.03" />
      <param name="circ_point" value = "1000" />
      <param name="resolution" value="0.001" />
      <param name="distance_approach" value="0.02" />
      <param name="ctrl_pt_1" value = "0.05/" />
      <param name="ctrl_pt_2" value = "0.05" />
      <param name="corner_points" value = "100" />
      <param name="fix_init_dist" value = "0.02" />
      <param name="fix_final_dist" value = "0.01" />
      <param name="corner_points" value = "1000" />
      <param name="cornerfix_init_dist" value = "0.02" />
      <param name="cornerfix_final_dist" value = "0.03" />
      <param name="cornerfix_lateral_schift" value = "0.03" />
      <param name="cornerfix_round_height" value = "0.03"/>

	</node>

</launch>